### DISCUSSION TITLE
SemanticCommitType setting is ignored

### DISCUSSION THREAD

**Discussion by nosajsnikta** [renovatebot/renovate#8440] [Request Help] [answered]
I am trying to get renovate to use semantic commits so that any PRs raised will be prefixed with 'fix(deps):'

The only way I am currently able to do this is using the following config setting in my renovate.json file:

`"commitMessagePrefix": "fix(deps):"`

This seems like a bodge, as I think the following is more suitable:

`semanticCommitType: "fix"`

however, this always prefixes with 'chore(deps):' implying that the setting is being ignored. I don't believe I need to enable semanticCommits as the default of 'auto' should be sufficient.

Can anyone point me in the right direction? I'm using renovate version 22.25.0 if that helps

**Comment by rarkins** [comment, marked as answer]
If you're extending `config:base` then it includes this preset setting semantic prefixes: https://docs.renovatebot.com/presets-default/#semanticprefixfixdepschoreothers

Try adding `":semanticCommitTypeAll(fix)"` to your `extends` array to override it.

**Comment by nosajsnikta** [comment]
Thanks for your reply

My config.js does extend config:base so I can try what you said.

Out of interest is it possible to have an 'extends' in config.js and then a
further 'extends' in the project renovate.json?

I did notice that the docs said you cannot define your extends array in a
config.js also so if you can advise there that would be great.

Thanks

**Reply by rarkins** [reply]
You can have both. We do support extends in config.js now but recommend they're instead in the repo config.

**Comment by nosajsnikta** [comment]
Ok thanks for the tips all working well now.

Thanks
